gatsby-config.js

```diff
@@ -56,24 +60,16 @@ module.exports = {
     {
       resolve: 'gatsby-source-filesystem',
       options: {
-        path: `${__dirname}/content/meta/`,
```

Comment:
this was causing the build error because `meta` folder didn't exist. a good way to test things out is to run `gatsby build`.

Reply:
🙌Thanks for the fix!

Good to know – have never understood what 'build' functions do that's different to 'develop'